### PR TITLE
Docs: Add Storybook docs for BigValue

### PR DESCRIPTION
**Pull request by peterholmberg**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Part of #25513

**Special notes for your reviewer**:

### CODE REVIEW

**Review by Clarity-89 — APPROVED** [review]

**Review by hugohaggmark — APPROVED** [review]
LGTM!

File: packages/grafana-ui/src/components/BigValue/BigValue.mdx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import { BigValue } from './BigValue';
+
+<Meta title="MDX|BigValue" component={BigValue} />
+
+# BigValue
+Component for showing a value based on a [DisplayValue](https://grafana.com/docs/grafana/latest/packages_api/data/displayvalue/#displayvalue-interface).
```
Comment (jessover9000) [inline packages/grafana-ui/src/components/BigValue/BigValue.mdx:7]: I suggest adding some info about that optional (if optional? after what we discussed in the weekly about the sparkline being hard-coded I'm not sure :D) sparkline. Also the text mode part of being able to change between showing a name, value or both could be interesting. The text mode select also has "auto" – what exactly does auto mean, do we know?

**Review by jessover9000 — COMMENTED** [review]
Good start! Love that you added examples 🚀 💯  Added some suggestions to make the docs even better.

**Review by mckn — APPROVED** [review]
Boooom! Great work @peterholmberg!

**Review by jessover9000 — APPROVED** [review]
Looks great now!

**Pull request merged**